invenio_requests/assets/semantic-ui/js/invenio_requests/confirmationModal/state/actions.js

```diff
@@ -0,0 +1,35 @@
+import { errorSerializer } from "../../api/serializers";
+
+export const OPEN_MODAL = "confirmation_modal/OPEN_MODAL";
+export const CLOSE_MODAL = "confirmation_modal/CLOSE_MODAL";
+export const START_ACTION = "confirmation_modal/START_ACTION";
+export const ACTION_SUCCESS = "confirmation_modal/ACTION_SUCCESS";
+export const HAS_ERROR = "confirmation_modal/HAS_ERROR";
+
+export const openModal = ({ text, action }) => {
+  return async (dispatch) => {
+    dispatch({ type: OPEN_MODAL, payload: { text, action } });
+  };
+};
+
+export const startAction = () => {
+  return async (dispatch, getState) => {
```

Comment:
```suggestion
  return async (dispatch, getState, config) => {
```
we have extra arguments injected to the store (see store.js) I am not sure if not including it in the action signature won't break anything. to be tested.

Reply:
noted :+1:

Reply:
It is ok without, fortunately this JS :)